presto-docs/src/main/sphinx/release/release-340.md

```diff
@@ -0,0 +1,60 @@
+# Release 340 (XX Aug 2020)
+
+## General Changes
+
+* Add support for query parameters in `LIMIT`, `OFFSET` and `FETCH FIRST` clauses. ({issue}`4529`, {issue}`4601`)
+* Add {func}`bitwise_left_shift`, {func}`bitwise_right_shift` and {func}`bitwise_right_shift_arithmetic`. ({issue}`772`)
+* Improve dynamic partition pruning for short queries. ({issue}`4388`}
+* Reduce cluster load by cancelling query stages from which data is no longer required. ({issue}`4290`)
+* Reduce query memory usage by improving retained size estimation for `VARCHAR` and `CHAR` types. ({issue}`4123`)
+* Improve query performance for queries containing ``starts_with`` predicates. ({issue}`4669`)
+* Fix failure when `GROUP BY` clause contains duplicate expressions. ({issue}`4609`)
+* Fix potential hang during query planning ({issue}`4635`).
+
+## Security
+
+* Verify access to referenced table in `CREATE TABLE LIKE`. ({issue}`4472`)
+
+## JDBC Driver Changes
+
+* Fix handling of dates and timestamps before the 20th century. ({issue}`4563`)
+
+## Elasticsearch Connector Changes
+
+* Fix failure when index mapping is missing. ({issue}`4535`)
+
+## Hive Connector Changes
+
+* Improve performance when querying tables with `skip.header.line.count` property set to 1. ({issue}`4513`)
+* Allow creating a table with `external_location` when schema's location is not valid. ({issue}`4069`)
+* Add read support for tables that were created as non-transactional and converted to be
+  transactional later. ({issue}`2293`)
+* Allow creation of transactional tables. Note: inserts to transactional tables
```

Comment:
```suggestion
* Allow `CREATE TABLE` for transactional tables. Note: inserts to transactional tables
```

Reply:
The original is fine. It's explaining a concept, not specific SQL syntax.